### PR TITLE
[12.0][MIG] base_cron_exclusion

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Superseed #1393 due to get no answer on https://github.com/OCA/server-tools/pull/1393#issuecomment-494025547

CC @Eficent

### CODE REVIEW

File: base_cron_exclusion/views/ir_cron_view.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+
+<odoo>
+    <record id="ir_cron_view" model="ir.ui.view">
```
Comment (tbaden) [inline base_cron_exclusion/views/ir_cron_view.xml]: optional change:
use original id (ir_cron_view_form)

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM

File: base_cron_exclusion/views/ir_cron_view.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply base_cron_exclusion/views/ir_cron_view.xml]: done!

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by AdriaGForgeFlow — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM

**Pull request closed without merging**